### PR TITLE
fix: font rendered different on some sites

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- We adapted font-rendering from each website, but we wanted to render smoothly
- This PR introduces the same font rendering for all websites so we uniform our companion fonts.

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-108 #done

### CODE REVIEW

File: packages/extension/public/css/daily-companion-app.css [hunk 1]
```diff
@@ -8,4 +8,7 @@ daily-companion-app {
   position: fixed;
   z-index: 2147483647;
   font-feature-settings: normal!important;
```
Comment (kirillkurko) [inline packages/extension/public/css/daily-companion-app.css:10]: I'm going to work as a prettier here - let's insert a whitespace before !important  😀

**Review by kirillkurko — APPROVED** [review]
Just a minor comment

File: packages/extension/public/css/daily-companion-app.css (hunk 1 above)
Comment (rebelchris) [reply packages/extension/public/css/daily-companion-app.css]: I think both variants are valid, but added a space 👍

**Review by sshanzel — APPROVED** [review]
Whoa! So nice you found the fix for this! 🚢 🚢 🚢

**Pull request merged**